python-checks/src/main/java/org/sonar/python/checks/ChildAndParentExceptionCaughtCheck.java

```diff
@@ -132,7 +132,7 @@ private static PythonQuickFix createQuickFix(Expression currentException) {
         .map(ExceptClause.class::cast)
         .map(ExceptClause::exception)
         .map(exceptions -> {
-          List<String> names = collectNamesFromTuple(exceptions);
+          List<String> names = new ArrayList<>(collectNamesFromTuple(exceptions));
```

Comment:
In this case, wouldn't it be better to keep `Collectors.toList()` and flag the issue as FP?
It feels a bit convoluted to create an immutable list which is only be used to re-create a mutable one.

Reply:
Yes, let's do that